### PR TITLE
Update weekly test list

### LINKED ISSUE #2192: Add additional test targets for arm32
We [recently enabled](https://github.com/AdoptOpenJDK/openjdk-build/pull/2104) sanity.openjdk to this platform now that:
- we have test machine capacity
- there is a JIT so tests will run in a timely fashion 

Now we can further expand testing on this platform (to include system and perf tests).

### PR DESCRIPTION
**Pull request by smlambert**
Updated weekly test list and add 'default' set to jdk8 arm32 and jdk11 aarch64
Fixes #2192 and partially #2189

### CODE REVIEW

**Comment by karianna** [comment]
@smlambert conflicts

**Comment by smlambert** [comment]
Ah yes, the joy of following a change to 'fix' the formatting but with no functional change enclosed.

**Review by sxa — APPROVED** [review]
Woohoo!

**Comment by sxa** [comment]
@karianna Since you commented earlier are you willing to review this?

File: pipelines/build/common/build_base_file.groovy [hunk 1]
```diff
@@ -61,23 +61,17 @@ class Builder implements Serializable {
...
-        'sanity.external'
+        'sanity.functional',
+        'extended.functional'
     ]
-    
-    // Temporarily remove weekly tests due to lack of machine capacity during release
-    final List<String> weekly = []
-    /*
+
     final List<String> weekly = [
```
Comment (aahlenst) [inline pipelines/build/common/build_base_file.groovy:68]: Can we get a short explanation or link about the test selection? Will daily and weekly be combined during release? Will both run on a weekend?

Comment (smlambert) [reply pipelines/build/common/build_base_file.groovy:75]: I've pushed some comments - and will update the "Graduated Testing wiki, once we actually schedule a 1x/week weekly build pipeline

**Review by aahlenst — APPROVED** [review]

File: pipelines/build/common/build_base_file.groovy (hunk 1 above)
Comment (aahlenst) [reply pipelines/build/common/build_base_file.groovy:75]: Thanks!

**Pull request merged**